lib/SILOptimizer/Utils/Generics.cpp

```diff
@@ -525,6 +525,15 @@ bool ReabstractionInfo::prepareAndCheck(ApplySite Apply, SILFunction *Callee,
     return false;
   }
 
+  // SWIFT_ENABLE_TENSORFLOW
+  // Disable specialization for instructions that are operands of
```

Comment:
This change is necessary to prevent post-differentiation SIL passes from generating invalid SIL.

---

In particular, this change disables specialization of operands to `autodiff_function`.
It may be possible to teach the specializer how to handle `autodiff_function` instructions. I don't know much work that would take, it seems difficult.